generators/server/templates/src/main/java/package/config/ApplicationProperties.java.ejs

```diff
@@ -28,5 +28,21 @@ import org.springframework.boot.context.properties.ConfigurationProperties;
  */
 @ConfigurationProperties(prefix = "application", ignoreUnknownFields = false)
 public class ApplicationProperties {
+    private final AuditEvents auditEvents = new AuditEvents();
```

Comment:
Those configuration should probably be moved to jhispter project: https://github.com/jhipster/jhipster/blob/master/jhipster-framework/src/main/java/io/github/jhipster/config/JHipsterProperties.java

Reply:
I was thinking the same. But then thought that `ApplicationProperties` class is always empty and this property sample can work as a way to reference. But also not a big deal to keep or move to the jar.

Reply:
ApplicationProperties class is always empty because we just want to add client business configuration...

Reply:
I agree with you. From the jhipster platform point of view, it should reside inside platform provided jar.

Reply:
@DanielFran : Thanks. Done 😄